app/models/concerns/host_common.rb

```diff
@@ -15,14 +15,14 @@ module HostCommon
   included do
     belongs_to_proxy :puppet_proxy,
       :feature => N_('Puppet'),
-      :label => N_('Puppet Master'),
-      :description => N_('Use this puppet server as an initial Puppet Server or to execute puppet runs'),
+      :label => N_('Puppet Proxy'),
+      :description => N_('Enter the default Smart Proxy used to access the Puppetserver API'),
```

Comment:
To me default implies there's a fallback so I'd leave out `default`.

But thinking more, how about something like:
```suggestion
      :description => N_('Use the Puppetserver configured on this Smart Proxy.'),
```

Reply:
I like that better, adjusted